src/System.Management.Automation/resources/RunspaceInit.resx

```diff
@@ -186,6 +186,9 @@
   <data name="NestedPromptLevelDescription" xml:space="preserve">
     <value>Dictates what type of prompt should be displayed for the current nesting level</value>
   </data>
+  <data name="PSNativeCommandUseErrorActionPreferenceDescription" xml:space="preserve">
+    <value>If true, $ErrorActionPreference applies to native executables, so that non-zero exit codes will generate cmdlet-style errors governed by error action settings</value>
```

Comment:
Probably need:
```suggestion
    <value>If true, $ErrorActionPreference applies to native executables, so that non-zero exit codes will generate cmdlet-style errors governed by error action settings.</value>
```

Reply:
None of the other messages here end with `.`

Reply:
There are both cases. We need use right. But I don't know what case is right :-)